### PR TITLE
2 3 1 release notes

### PR DESCRIPTION
**Pull request by nathan-px**

### CODE REVIEW

File: content/reference/release-notes/portworx.md [hunk 1]
```diff
@@ -6,6 +6,34 @@ keywords: portworx, release notes
...
+### New Features
+
+*
+-->
+### Improvements
+
+Portworx has upgraded or enhanced functionality in the following areas:
+
+|**Improvement Number**|**Improvement Description**|
+|----|----|
```
Comment (vilasd) [inline content/reference/release-notes/portworx.md]: We need to call out support for AWS and azure cloud providers

Comment (nathan-px) [reply content/reference/release-notes/portworx.md]: For line 23? Is disk resize supported on both?

File: content/reference/release-notes/portworx.md [hunk 2]
```diff
@@ -6,6 +6,34 @@ keywords: portworx, release notes
...
+| PWX-10148 | If you're using Portworx on Microsoft Azure, Portworx can now expand storage pools by resizing disks. |
+| PWX-10332 | Portworx now provides more descriptive error messages for pool expansion failures. |
+| PWX-10442 | The `pxctl service pool expand` command is now available, allowing you to expand storage pools by adding drives and consuming unused drive capacity. <!-- do we want to present this as a new feature? -->
```
Comment (nathan-px) [inline content/reference/release-notes/portworx.md]: @harsh-px You listed a number of release notes under a single issue, so I broke them out in the list here and just duplicated the number: 10442.

**Comment by nathan-px** [comment]
@harsh-px , added you in case you want to take a look. I think a lot of the issues were yours.

File: content/reference/release-notes/portworx.md (hunk 2 above)
Comment (harsh-px) [reply content/reference/release-notes/portworx.md]: Yes, this is a new feature

File: content/reference/release-notes/portworx.md [hunk 3]
```diff
@@ -6,6 +6,34 @@ keywords: portworx, release notes
...
+| PWX-10442 | Added support to list volumes per pool UUID <!-- where? need more information on this --> |
```
Comment (harsh-px) [inline content/reference/release-notes/portworx.md]: This is a new flag in the volume list command.

`pxctl volume list --pool-uid <uuid>`

File: content/reference/release-notes/portworx.md [hunk 4]
```diff
@@ -6,6 +6,34 @@ keywords: portworx, release notes
...
+
+### Fixes
+
+The following issues have been fixed:
+
+|**Issue Number**|**Issue Description**|
+|----|----|
+| PWX-10414 | Storage pools sometimes fail to come back online after a disk is added as part of a pool expand operation.<br/><br/>**User Impact:** Impacted storage pools may remain down, impacting apps. <!-- What caused this? Did this occur only with autopilot? Was it intermittent? --> |
+| PWX-10442 | Autopilot, or users entering the `pxctl service pool expand` command, can incorrectly start pool resize operations when pool volumes are undergoing resync operations.<br/><br/>**User Impact:** These pool resize operations fail. |
```
Comment (harsh-px) [inline content/reference/release-notes/portworx.md]: We need to clarify that this issue is now fixed

Comment (harsh-px) [inline content/reference/release-notes/portworx.md]: @vilasd we had not exposed `pxctl sv pool expand` in 2.3.0. The CLI was hidden. So do you still need to release note fixes that went in?

Comment (vilasd) [reply content/reference/release-notes/portworx.md]: @harsh-px pxctl command for pool expand is not hidden anymore in 2.3.1 so we are good to release note this

Comment (vilasd) [inline content/reference/release-notes/portworx.md]: We don't need to release note PWX-10442

**Pull request merged**